wye/workshops/forms.py

```diff
@@ -3,7 +3,7 @@
 
 from wye.base.widgets import CalendarWidget
 
-from .models import Workshop
+from . import models
```

Comment:
We should have guidelines (sometime soon). Inside apple we can use relative imports directly imported from models. Reason is `models` name collides with `Django models`.